Automattic/wp-calypso
# desktop: use calypso.live for desktop e2e

### Description

- Add support for specifying the calypso URL. This can be useful for the purposes of pointing desktop CI to the `calypso.live` for a given branch and PR.
- Pass the calypso.live URL to the desktop e2e runner.

To Test:

- Pick a calypso.live URL and try running the application with `WP_DESKTOP_BASE_URL=<URL> yarn run dev` from the `desktop` folder of the repository.
- Desktop e2e build should pass.
